cmd/agent/app/builder.go

```diff
@@ -39,6 +39,7 @@ import (
 const (
 	defaultQueueSize     = 1000
 	defaultMaxPacketSize = 65000
+	defaultBufferSize    = 4 * 1024
```

Comment:
this is declared in `transport.go` could we consume it from there?

Reply:
I think we should omit default value. If it's not explicitly set then agent must use system value.